### PR TITLE
Search bar detects integers and shows base 2 representations as l's and o's

### PR DESCRIPTION
**Pull request by sandlerben**

### CODE REVIEW

**Comment by sandlerben** [comment]
r @weronica @riyer15

File: pennapps-16sp/ViewController.swift [hunk 1]
```diff
@@ -43,7 +43,21 @@ class ViewController: UIViewController {
 extension ViewController: UISearchBarDelegate {
     
     func searchBarSearchButtonClicked(searchBar: UISearchBar) {
-        print("Search button clicked")
+        let text = searchBar.text
+        if let number = Int(text!)
+        {
```
Comment (skylerwharton) [inline pennapps-16sp/ViewController.swift]: put `{` on same line please

File: pennapps-16sp/ViewController.swift [hunk 2]
```diff
@@ -43,7 +43,21 @@ class ViewController: UIViewController {
...
+            showBinaryPopup(number);
+        }
+        else
+        {
```
Comment (skylerwharton) [inline pennapps-16sp/ViewController.swift]: Same

File: pennapps-16sp/ViewController.swift [hunk 3]
```diff
@@ -43,7 +43,21 @@ class ViewController: UIViewController {
 extension ViewController: UISearchBarDelegate {
     
     func searchBarSearchButtonClicked(searchBar: UISearchBar) {
-        print("Search button clicked")
+        let text = searchBar.text
+        if let number = Int(text!)
+        {
+            showBinaryPopup(number);
```
Comment (skylerwharton) [inline pennapps-16sp/ViewController.swift]: Remove `;`

File: pennapps-16sp/ViewController.swift (hunk 1 above)
Comment (sandlerben) [inline pennapps-16sp/ViewController.swift]: LOL

File: pennapps-16sp/ViewController.swift (hunk 3 above)
Comment (sandlerben) [inline pennapps-16sp/ViewController.swift]: LOL

File: pennapps-16sp/ViewController.swift (hunk 2 above)
Comment (sandlerben) [inline pennapps-16sp/ViewController.swift]: LOL

File: pennapps-16sp/ViewController.swift [hunk 4]
```diff
@@ -43,7 +43,21 @@ class ViewController: UIViewController {
...
+        }
+        else
+        {
+            print("value: \(text) is not a valid  number.")
+        }
+    }
+    
+    func showBinaryPopup(number: Int) {
```
Comment (skylerwharton) [inline pennapps-16sp/ViewController.swift]: `showBinaryAlert`

**Pull request merged**